src/styles/global.css

```
@@ -87,6 +89,11 @@ html.dark {
   color: #fff;
 }
 
+.dark .nav-links a:hover,
+.dark .nav-links a:focus {
+  color: #000;

```

```suggestion
  color: #0d0950;
```

We thought the dark mode dark blue background color was a better choice for the hovered/focused link text color.